SoftLayer/managers/firewall.py

```diff
@@ -302,3 +302,19 @@ def get_instance(self, firewall_id, mask=None):
         svc = self.client['Network_Vlan_Firewall']
 
         return svc.getObject(id=firewall_id, mask=mask)
+
+    def get_firewalls_gatewalls(self):
+        """Returns a list of all firewalls on the account.
+
+        returns: A list of firewalls on the current account.
+        """
```

Comment:
I dont mean to nit-pick here, but `gatewall` isn't an English word. However as a mashup of "gateway" + "firewall" I do kinda like it, and want to keep it. Just make the following change to the docblock so anyone else knows you put `gatewalls` on purpose, and just didn't misspell `gateways`, which is what I thought when I first saw it lol.

```suggestion
    def get_firewalls_gatewalls(self):
        """Returns a list of all gateway firewalls (gatewalls) on the account.

        returns: A list of gateway firewalls (gatewalls) on the current account.
        """
```